app/hooks/use-mempool.ts

```diff
@@ -26,7 +26,7 @@ export function useMempool() {
   const mempoolFetcher = useCallback(() => new Api(activeNode.url).getMempoolTransactions(), [
     activeNode,
   ]);
-  const { data: mempoolTxs } = useSWR('mempool', mempoolFetcher, { refreshInterval: 1_500 });
+  const { data: mempoolTxs } = useSWR('mempool', mempoolFetcher, { refreshInterval: 10_000 });
```

Comment:
👍

Reply:
would recommend using `SWRConfig` at the root and make this happen only once, and then use a constant for it, eg `DEFAULT_POLLING_INTERVAL`